packages/shorebird_cli/lib/src/commands/preview_command.dart

```diff
@@ -148,14 +154,17 @@ class PreviewCommand extends ShorebirdCommand {
     String? deviceId,
   }) async {
     const platform = ReleasePlatform.android;
-    final aabPath = getArtifactPath(
-      appId: appId,
-      release: release,
-      platform: platform,
-      extension: 'aab',
+    final aabFile = File(
+      getArtifactPath(
+        appId: appId,
+        release: release,
+        platform: platform,
+        extension: 'aab',
+      ),
     );
 
-    if (!File(aabPath).existsSync()) {
+    if (!aabFile.existsSync()) {
+      aabFile.createSync(recursive: true);
```

Comment:
why is this needed? Shouldn't this be handled by artifact manager's downloadFile?

Reply:
Good call, fixed.